week01/BestTimetoBuyandSellStock.cpp

```diff
@@ -0,0 +1,23 @@
+class Solution {
+public:
+    int maxProfit(vector<int>& prices) {
+        int siz = prices.size();
+        int prefixMin[siz + 5], suffixMax[siz + 5];
```

Comment:
This is just plain BAD. https://stackoverflow.com/questions/24136120/what-is-wrong-with-using-arrays-dynamically-allocated-in-c